### PR TITLE
Use private header import for FIRVersion.h

### PR DESCRIPTION
**Pull request by ryanwilson**
This was causing a failure in publishing the RemoteConfig pod.

#no-changelog

### CODE REVIEW

File: FirebaseRemoteConfig/Sources/RCNDevice.m [hunk 1]
```diff
@@ -19,9 +19,9 @@
 #import <sys/utsname.h>
 
 #import <GoogleUtilities/GULAppEnvironmentUtil.h>
-#import "FirebaseCore/Sources/Public/FirebaseCore/FIRVersion.h"
 #import "FirebaseRemoteConfig/Sources/Private/RCNConfigSettings.h"
 #import "FirebaseRemoteConfig/Sources/RCNConfigConstants.h"
+#import "FIRVersion.h"
```
Comment (maksymmalyhin) [inline FirebaseRemoteConfig/Sources/RCNDevice.m:24]: I think it should be either
```
#import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
```
or
```
#import <FirebaseCore/FirebaseCore.h>
```

Comment (paulb777) [reply FirebaseRemoteConfig/Sources/RCNDevice.m:24]: Yes, either should work, but the normal pattern in the repo is the former.

Comment (ryanwilson) [reply FirebaseRemoteConfig/Sources/RCNDevice.m:24]: I misread the docs! I caught the second half of the exception but not the first. I had the first suggestion you listed before but then thought I followed the right rule... whoops :) nice catch, thanks.

**Review by maksymmalyhin — APPROVED** [review]

**Review by paulb777 — APPROVED** [review]

**Comment by ryanwilson** [comment]
cc @karenyz as an FYI

**Comment by karenyz** [comment]
oh sorry about that!

**Comment by ryanwilson** [comment]
> oh sorry about that!

No problem! We should have the tooling that catches this - it passed both review and CI so that's definitely not on you 😄

**Pull request merged**